src/components/study-groups/StudyGroupMarketplace.tsx

```
@@ -0,0 +1,183 @@
+                <div className="max-w-7xl mx-auto px-4 sm:px-6 py-5 flex items-center justify-between">
+                    <div className="flex items-center gap-4">
+                        <div className="w-12 h-12 rounded-xl bg-gradient-to-br from-indigo-500 to-rose-600 flex items-center justify-center shadow-lg shadow-indigo-500/20 border border-indigo-400/30">
+                            <Users className="w-6 h-6 text-white" />
+                        </div>
+                        <div>
+                            <h1 className="text-xl font-black text-white leading-tight">Live Study Groups</h1>
+                            <p className="text-xs tracking-wider text-slate-400 font-bold uppercase mt-1">Cram together. Pass together.</p>
+                        </div>
+                    </div>
+
+                    <button className="bg-indigo-500 hover:bg-indigo-400 text-white font-bold py-2.5 px-6 rounded-xl transition-all shadow-lg flex-center items-center gap-2 text-sm shadow-indigo-500/20">
```

ESLint

[error] Delete `··········`

(prettier/prettier)